mailet/standard/src/test/java/org/apache/james/transport/mailets/ContactExtractorTest.java

```diff
@@ -183,6 +183,24 @@ public void serviceShouldSupportMultipleFromField() throws Exception {
     }
 
     @Test
+    public void serviceShouldSkipMessagesWithoutFromAndSender() throws Exception {
+        MimeMessage message = MimeMessageBuilder.mimeMessageBuilder()
+            .addToRecipient("To <" + TO + ">")
+            .setSubject("Contact collection Rocks")
+            .setText("This is my email")
+            .build();
+        FakeMail mail = FakeMail.builder().mimeMessage(message)
+            .sender(new MailAddress(SENDER))
+            .recipient(new MailAddress(TO))
+            .build();
+        mailet.init(mailetConfig);
+
+        mailet.service(mail);
+
+        assertThatJson(mail.getAttribute(ATTRIBUTE)).isEqualTo(null);
```

Comment:
The value of the attribute is null or the attribute is not present?

Reply:
Null means not present.

Here there is also some concerns with assertj on serializable stull.

Maybe we need to assert attributes names does not contain Blah...

Reply:
Oh crap. That's why I thought assertj was limited with serializable....

Face palm